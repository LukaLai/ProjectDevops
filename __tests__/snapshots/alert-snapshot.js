import { render } from '@testing-library/react'
import Alert from '../../src/app/_components/alert'

it('renders homepage unchanged', () => {
    const { container } = render(<Alert />)
    expect(container).toMatchSnapshot()
})