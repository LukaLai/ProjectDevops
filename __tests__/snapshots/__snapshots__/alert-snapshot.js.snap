// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="border-b bg-neutral-50 border-neutral-200"
  >
    <div
      class="container mx-auto px-5"
    >
      <div
        class="py-2 text-center text-sm"
      >
        The source code for this blog is
         
        <a
          class="underline hover:text-blue-600 duration-200 transition-colors"
          href="https://github.com/vercel/next.js/tree/canary/examples/blog-starter"
        >
          available on GitHub
        </a>
        .
      </div>
    </div>
  </div>
</div>
`;
